hmda/src/main/resources/hmda.conf

```diff
@@ -40,6 +40,8 @@ hmda {
     shardNumber = 100
   }
 
+  pageSize = 20
```

Comment:
I would add an environment variable here so that we can change this value at runtime without having to recompile